tests/objvariant/trt_discrim_err2.nim

```diff
@@ -1,10 +1,10 @@
 discard """
-  errormsg: "low(kind) must be 0 for discriminant"
-  line: 7
+  errormsg: " branch initialization with a runtime discriminator only supports ordinal types with 2^16 elements or less."
```

Comment:
We should add this as an extra test case.

Reply:
Did you see not tnon_zero_discrim_err.nim or do you mean something else?

Reply:
Oh, yeah somehow I missed that, sorry